administration/src/components/NotFoundView.tsx

```diff
@@ -0,0 +1,13 @@
+import { NonIdealState } from '@blueprintjs/core'
+import styled from 'styled-components'
+
+const NotFound = styled(NonIdealState)`
+  margin: auto;
+`
+const NotFoundView = () => {
+  return (
+      <NotFound title='Nicht gefunden' description='Diese Seite konnte nicht gefunden werden.' />
```

Comment:
i think the message should be a little bit more specific .f.e "Es konnte kein Antrag gefunden werden". If you create a separate component which means it can be reused i would tend to pass title and description or just use the `<NotFound>` directly in `VerificationController`

Reply:
I moved this one into the VerificationController for now.